src/Components/Custom-Button/index.js

```diff
@@ -1,4 +1,20 @@
 import React from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+
+const button = styled.button`
+  color: #aa3939;
+  font-size: 1em;
```

Comment:
- [ ] For this project we are going to use `rem` instead of `em` and `px`. Can you change you make sure we are using rems everywhere?